package.json

```diff
@@ -43,7 +43,8 @@
     "puppeteer": "1.20.0",
     "rollup-plugin-node-polyfills": "^0.2.1",
     "ts-node": "^8.5.0",
-    "typescript": "^3.7.5"
+    "typescript": "^3.8.3",
+    "workbox-build": "4.3.1"
```

Comment:
what is `workbox-build`?

Reply:
Used by Stencil

Reply:
Like installed by Stencil during the build?